cli/kill.go

```diff
@@ -103,9 +103,9 @@ func kill(containerID, signal string, all bool) error {
 		return err
 	}
 
-	// container MUST be created or running
-	if status.State.State != vc.StateReady && status.State.State != vc.StateRunning {
-		return fmt.Errorf("Container %s not ready or running, cannot send a signal", containerID)
+	// container MUST be created, running or paused
+	if status.State.State != vc.StateReady && status.State.State != vc.StateRunning && status.State.State != vc.StatePaused {
```

Comment:
I'd say this is ok, but it's on the verge of needing to be turned into a switch.

/me waits for laughter from @sboeuf for the use of "that word" ;)

Reply:
Yeah could be a switch but I am fine with it ;)

Reply:
@sboeuf - I was referring to the amusing translation of the "v word" ;)